File: pbworkspacewcl/src/app/attivita/commons/vo/progetto-vo.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { AttivitaVO } from './attivita-vo';

export class ProgettoVO {
    constructor(
        public id: number,
        public acronimo: string,
        public codiceVisualizzato: string,
        public titolo: string,
        public attivita: Array<AttivitaVO>
    ) { }
}

File: pbgestfinbo/pbgestfinbo-jar/src/main/java/it/csi/pbandi/pbgestfinbo/integration/vo/controdeduzioni/ProgettoVO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbgestfinbo.integration.vo.controdeduzioni;

public class ProgettoVO {
    private Long idProgetto;
    private String codiceVisualizzatoProgetto;
    private String titoloProgetto;

    public Long getIdProgetto() {
        return idProgetto;
    }

    public void setIdProgetto(Long idProgetto) {
        this.idProgetto = idProgetto;
    }

    public String getCodiceVisualizzatoProgetto() {
        return codiceVisualizzatoProgetto;
    }

    public void setCodiceVisualizzatoProgetto(String codiceVisualizzatoProgetto) {
        this.codiceVisualizzatoProgetto = codiceVisualizzatoProgetto;
    }

    public String getTitoloProgetto() {
        return titoloProgetto;
    }

    public void setTitoloProgetto(String titoloProgetto) {
        this.titoloProgetto = titoloProgetto;
    }
}
